x-pack/legacy/plugins/siem/public/components/embeddables/map_tool_tip/tooltip_footer.test.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { ToolTipFooter } from './tooltip_footer';
+
+describe('ToolTipFilter', () => {
+  let nextFeature = jest.fn();
+  let previousFeature = jest.fn();
+
+  beforeEach(() => {
+    nextFeature = jest.fn();
+    previousFeature = jest.fn();
+  });
+
+  test('renders correctly against snapshot', () => {
+    const wrapper = shallow(
+      <ToolTipFooter
+        nextFeature={nextFeature}
+        previousFeature={previousFeature}
+        featureIndex={0}
+        totalFeatures={100}
+      />
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  test('previousButton is functioning and nextButton is not when featureIndex is 0', () => {
```

Comment:
should this read something like `previousButton is disabled and nextButton is functioning when featureIndex is 0`?

Reply:
consider structuring the tests in this file such that there is one `expect` per test

Reply:
Fixed wording to be less ambiguous as part of structuring tests to only have one expect each. Also bundled in appropriately named groups as well. Test structure is as follows:

![image](https://user-images.githubusercontent.com/2946766/66356792-9688f600-e929-11e9-8133-cf76249845cb.png)